docs/lantern.md

```diff
@@ -0,0 +1,32 @@
+# Lantern
+
+## Overview
+
+Project Lantern is an ongoing effort to reduce the run time of Lighthouse and improve audit quality by modeling page activity and simulating browser execution.
+
+## Accuracy
+
+All of the following accuracy stats are reported excluding the 10% tail as the initial research found approximately ~10% of sites will radically vary simply by visiting the page a second time through no fault of the metrics or prediction logic. This means the accuracy is slightly overstated but should still hold for the  controlled-enivornment/repeated view use case.
```

Comment:
Can we make it explicit that this was calculated based on an analysis of 1500 URLs run only *once*

Reply:
done